java/clients/src/test/java/sleeper/clients/admin/testutils/AdminClientITBase.java

```diff
@@ -51,6 +53,10 @@ public abstract class AdminClientITBase extends AdminClientTestBase {
             .withEndpointConfiguration(localStackContainer.getEndpointConfiguration(LocalStackContainer.Service.S3))
             .withCredentials(localStackContainer.getDefaultCredentialsProvider())
             .build();
+    protected final AmazonSQS sqs = AmazonSQSClientBuilder.standard()
+            .withCredentials(localStackContainer.getDefaultCredentialsProvider())
+            .withEndpointConfiguration(localStackContainer.getEndpointConfiguration(LocalStackContainer.Service.DYNAMODB))
+            .build();
```

Comment:
This is pointing the SQS client to the DynamoDB endpoint. DynamoDB and SQS are both disabled in LocalStack, so we can either remove this completely or add an integration test for it.